src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -521,6 +521,19 @@ private function getColumnWidth($column)
                 continue;
             }
 
+            // split the TableCell centent in order to extending width of other columns
```

Comment:
I don't understand what does this comment try to explain. We extend this cell up to the width of the largest column?